### PR TITLE
Corrected subserver & added chdev/chfs

### PR DESCRIPTION
**Pull request by vfoucault**
This Pull request compile the previously created ones (and closed).

Fixed the `subserver` provider which wasn't working. It has some defects loading the `inetd.conf` resources and commands to create/update were wrong.
Corrected some issues also in the resource.

Added the following providers :

* `chdev` : udpate AIX devices attributes
* `chfs`: change AIX filesystem size (+/-)

updated the `README.md` to include some details about the providers.

### CODE REVIEW

File: resources/chdev.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+default_action :update
+
+attribute :name, :kind_of => String, :name_attribute => true
+attribute :attributes, :kind_of => Hash
+attribute :atreboot, :kind_of => [TrueClass, FalseClass], :default => true
+
+attr_accessor :exists
+
+
+# TODO:
```
Comment (juliandunn) [inline resources/chdev.rb]: Are there actually TODOs left?

File: resources/chfs.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+attribute :unit, :kind_of => String, :equal_to => ["G", "M", "K"], :default => "G"
+attribute :sign, :kind_of => String, :equal_to => ["+", "-", ""]
+attribute :mountpoint, :kind_of => String
+attribute :options, :kind_of => [String, Array]
+attribute :automount, :kind_of => [TrueClass, FalseClass]
+
+attr_accessor :exists
+
+
+# TODO:
```
Comment (juliandunn) [inline resources/chfs.rb]: Same here, what todos are there?

File: resources/subserver.rb [hunk 3]
```diff
@@ -2,13 +2,13 @@
 
 attribute :servicename, :name_attribute => true, :kind_of => String
 attribute :type, :kind_of => String, :equal_to => %w(dgram stream sunrpc_udp sunrpc_tcp)
-attribute :protocol, :kind_of => String, :required => true, :equal_to => %w(tcp udp tcp6 udp6)
+attribute :protocol, :kind_of => String, :required => false, :equal_to => %w(tcp udp tcp6 udp6)
```
Comment (juliandunn) [inline resources/subserver.rb]: What's this change for?

File: resources/chfs.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+actions :update, :resize
+default_action :resize
+
```
Comment (juliandunn) [inline resources/chfs.rb]: Some of the mandatory attributes should be marked as such with `:required => true`.

**Comment by juliandunn** [comment]
@vfoucault are you still working on this?

**Comment by chmod666org** [comment]
@vfoucault chfs has to be included in a lvm/fs provider. We are working on it. chdev has been commited in another pull request. Can I close the pull request ? Or are you still working on it ?

**Pull request closed without merging**